include/zephyr/linker/ram-end.ld

```diff
@@ -9,6 +9,7 @@
 	LAST_RAM_ALIGN
 #endif
 	_image_ram_end = .;
+	_image_ram_used = _image_ram_start - _image_ram_end;
```

Comment:
this does not give you the used size unless your end address is lower than your start address

Reply:
and please use `size`, to be consistent with other symbol definitions.

Reply:
> this does not give you the used size unless your end address is lower than your start address

Thank you for catching this- the cast to `uint32_t` was masking this for me when verifying the size calculation was right.